locked-asset/lkmex-transfer/src/lib.rs

```diff
@@ -46,6 +46,11 @@ pub trait LkmexTransfer:
     #[endpoint]
     fn withdraw(&self, sender: ManagedAddress) {
         let receiver = self.blockchain().get_caller();
+        require!(
+            !self.blockchain().is_smart_contract(&sender)
+                && !self.blockchain().is_smart_contract(&receiver),
+            "SCs cannot send or receive locked tokens"
+        );
```

Comment:
Not needed here. If nobody can deposit from/to SCs, then they'd have nothing to withdraw anyway.

Reply:
Removed.